.github/workflows/migrations.yml

```diff
@@ -1,99 +1,65 @@
-name: migrations
+name: generate migration sql
 on:
   pull_request:
     paths:
       - 'src/sentry/migrations/*'
 
 jobs:
-    sql:
-      name: Generate SQL
-      runs-on: ubuntu-16.04
+  task:
+    runs-on: ubuntu-16.04
 
-      env:
-        PIP_DISABLE_PIP_VERSION_CHECK: on
-        SENTRY_LIGHT_BUILD: 1
-        SENTRY_SKIP_BACKEND_VALIDATION: 1
-        MIGRATIONS_TEST_MIGRATE: 0
+    steps:
+      - uses: actions/checkout@v1
 
-        # The hostname used to communicate with the PostgreSQL from sentry
-        DATABASE_URL: postgresql://postgres:postgres@localhost/sentry
+      - name: Set python version output
+        id: python-version
+        run: |
+          echo "::set-output name=python-version::$(awk 'FNR == 2' .python-version)"
 
-      services:
-        postgres:
-          image: postgres:9.6
-          env:
-            POSTGRES_USER: postgres
-            POSTGRES_PASSWORD: postgres
-          ports:
-            # Maps tcp port 5432 on service container to the host
-            - 5432:5432
-          # needed because the postgres container does not provide a healthcheck
-          options: >-
-            --health-cmd pg_isready
-            --health-interval 10s
-            --health-timeout 5s
-            --health-retries 5
+      # Until GH composite actions can use `uses`, we need to setup python here
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ steps.python-version.outputs.python-version }}
 
-      steps:
-        - name: Install System Dependencies
-          run: |
-            sudo apt-get update
-            sudo apt-get install -y --no-install-recommends \
-              libxmlsec1-dev \
-              libmaxminddb-dev
+      - name: Setup sentry python env
+        uses: ./.github/actions/setup-sentry
+        id: setup
+        with:
+          python: 3
```

Comment:
This switches from 2 to 3, can we just keep this on 2 for now? I think that's what @wedamija wanted. Or, we run this on both for a while and see if they're consistent?